src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetApprenticePreferencesRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.SharedOuterApi.InnerApi.Requests
```

Comment:
So I know it was convention, but steps are being made to begin breaking out this APIM repository as it has become far too big. the split into folders with individual pipelines has been the start of that but it can't be split out because some of them share this Outer Api project.

I would argue that apprentice preferences is pretty specific, So i'd be tempted to move this back into specifically the apprentice feedback outer api project to not give us more tech debt later on.

You may also have issues with this PR now that the master branch has shifted under you, it can be rebased to work but you may find it easier to re-branch and reapply your changes depending on your preference